packages/e2e-test-app/windows/RNTesterApp/packages.lock.json

```diff
@@ -72,8 +72,8 @@
       },
       "ReactNative.Hermes.Windows": {
         "type": "Transitive",
-        "resolved": "0.70.0",
-        "contentHash": "nh9F1wAhkndqfHG/hrS+ezesunlGhCr32pASEEAkQenn8Ou4dUwjmt0c7aT3pONl9itZ1bF9GWbpXJ7t6AsS+Q=="
+        "resolved": "0.0.0-2209.12005-9f3ce7e5",
```

Comment:
> "0.0.0-2209.12005-9f3ce7e5", [](http://example.com/codeflow?start=19&length=29)

OOC, assuming this is not related to your change, what brings about modifications like this?

Reply:
Building those packages locally.
The CI builds don't make these updates, which are VS version-related (CI was recently updated to use version 2022).